en/sql/tuning.rst

```diff
@@ -649,6 +670,14 @@ The following hints can be specified in **UPDATE**, **DELETE** and **SELECT** st
     
         Hash aggregate evaluation will not work for functions evaluated on distinct values (e.g. AVG(DISTINCT x)) and for the GROUP_CONCAT and MEDIAN functions, since they require an extra sorting step for the tuples of each group.
 
+.. _no-hash-list-scan:
+
+*   **NO_HASH_LIST_SCAN**: This is a hint not to use hash list scan for scanning sub-query's result. Instead, list scan is used to scan temp file. By building and probing hash table, we can reduce the amount of data that needs to be searched. However, in some scenarios the user may know beforehand that outer cardinality is very small and can use the hint to skip hash list scan entirely. For setting the memory size of hash scan, see :ref:`max_hash_list_scan_size <max_hash_list_scan_size>`.
+
+    .. note::
+    
+        Hash List scan only work for predicates having a equal operation and do NOT work for predicate having oid type.
```

Comment:
Hash List scan only work for predicates having a equal operation and do NOT work for predicate having oid type.
 -->
Hash List scan only works for predicating having an equal operation and does NOT work for predicating having OID type.

Reply:
'Hash List scan only works for predicates having an equal operation and does NOT work for predicates having OID type.'
I think this is better. the 'predicate' was written as noun.